### ISSUE TITLE
[BUG] Cannot read properties of null (reading 'height')

### ISSUE THREAD

**Issue by hank0420** [Chia-Network/chia-blockchain-gui#2059]
### What happened?

cannot open 'farm'

### Version

2.0.0

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
Cannot read properties of null (reading 'height')

URL
#/dashboard/farm

Stacktrace
at webpack://@chia-network/gui/src/components/farm/FarmHealth.tsx:97:26 FarmHealth
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:157:136 Ch
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:180:153 oi
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:178:168 ki
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:177:177 ii
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:274:125 k.firstContext;null!==l;){if(l.context===d&&0!==(l.observedBits&f)){1===h.tag&&(l=zg(-1,c&-c),l.tag=2,Ag(h,l));h.lanes|=c;l=h.alternate;null!==l&&(l.lanes|=c);sg
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:250:346 Pj
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:250:277 Pj
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:250:137 Pj
at webpack://node_modules/react-dom/cjs/react-dom.production.min.js:243:162 Lj
```

**Comment by paninaro** [comment]
@ChiaMineJP Can you take a look at this issue? Looks like a simple fix to prevent the crash, but there may be something deeper to look into:

`height: blockchainState?.peak.height || 0,` --> `height: blockchainState?.peak?.height || 0,`

**Comment by ChiaMineJP** [comment]
@paninaro I've found typing issue. `get_blockchain_state` FullNode RPC API may return `peak: None` while initializing.
However, the typing for the API assumes that `peak` is always not `None`.
I've fixed the wrong typing as well as `height: blockchainState?.peak?.height || 0,`

**Comment by paninaro** [comment]
Fix will be in 2.1.0-rc3 or later.
